settings/src/client.rs

```diff
@@ -106,7 +106,7 @@ fn default_balance_notification() -> bool {
 #[derive(Debug, Serialize, Deserialize, Clone, Eq, PartialEq)]
 pub struct ExitClientSettings {
     /// This stores a mapping between an identifier (any string) to exits
-    #[serde(alias = "new_exits", default)]
+    #[serde(alias = "new_exits_temp", default)]
```

Comment:
this won't work, it needs to be just a "rename" with "new_exits". That will 'rename' the field when it loads it to just exits but keep it as new_exits in the config.

What you're doing here is interesting though, if we named our field new_exits and aliased to new exits tmp it might work. But I dont' think that's the way to go.